protocol-dashboard/src/components/TotalApiCallsChart/TotalApiCallsChart.tsx

```diff
@@ -3,15 +3,29 @@ import React, { useState } from 'react'
 import { useApiCalls } from 'store/cache/analytics/hooks'
 import { Bucket, MetricError } from 'store/cache/analytics/slice'
 import { datesToSkip } from 'utils/consts'
+import { useTrailingApiCalls } from 'store/cache/analytics/hooks'
 
 type OwnProps = {}
 
 type TotalApiCallsChartProps = OwnProps
 
-const TotalApiCallsChart: React.FC<TotalApiCallsChartProps> = props => {
-  const [bucket, setBucket] = useState(Bucket.MONTH)
+type ApiCallsBucket = Bucket.MONTH | Bucket.WEEK | Bucket.ALL_TIME | Bucket.YEAR
+
+const TotalApiCallsChart: React.FC<TotalApiCallsChartProps> = () => {
```

Comment:
no props any more right?

Reply:
yeah it was never used

Reply:
oh i see what you're saying yeah ill delete all this old stuff